### ISSUE TITLE
Page: FRNs

### ISSUE THREAD

**Issue by mcharg** [ONRR/onrr.gov-site#1250]
https://qmcq5k.axshare.com/#id=kkqoxq&p=federal_register_notices

**Comment by mcharg** [comment]
I was all excited that I found a page I could build completely, but it won't display because of the url bug.

**Comment by cthomasONRR** [comment]
@mcharg Sure- can I do this review tomorrow?

**Comment by mcharg** [comment]
@cthomasONRR Of course.

**Comment by cthomasONRR** [comment]
@mcharg Ok had some time to review while waiting on other projects. It looks good to me- the internal Rules&Regs and ICR webpage links send you to "page not found" but all the other links work. Thanks for pioneering the adding of content to CMS!

**Comment by mcharg** [comment]
@cthomasONRR Fixed the links.

**Comment by daniel-coleman-1** [comment]
@mcharg sentence case for those two

**Comment by mcharg** [comment]
@Maroyafaied I think we have our first completed page in the CMS.  Do you want to review?

**Comment by Maroyafaied** [comment]
@mcharg looks great, the only thing I noticed is the email link having ONRR underlined with it, not sure if we should only underline the mailbox or just take ONRR out. 
![image](https://user-images.githubusercontent.com/40175688/145588428-50ac3639-21ad-4af8-88d8-dd9624d99ac1.png)

**Comment by mcharg** [comment]
@Maroyafaied The ONRR is part of the email address: ONRR_RegulationsMailbox@onrr.gov

**Comment by mcharg** [comment]
When we get the email icon, it might help to put that in front of it.

**Comment by Maroyafaied** [comment]
@mcharg I didn't see the dash after ONRR.

**Comment by mcharg** [comment]
It's not showing up because of the underline.

**Comment by mcharg** [comment]
I'm going to move this back to the backlog, so we don't forget to add the email icon.

**Comment by mcharg** [comment]
@cthomasONRR This is ready for ODDD approval. Created a separate issue for email icons.

**Comment by cthomasONRR** [comment]
@mcharg Approved... I think we already got SME approval on this?

**Comment by mcharg** [comment]
@cthomasONRR Yes, we did get SME approval.
